pydicom/charset.py

```diff
@@ -95,6 +96,167 @@
                      'iso_ir_58')
 
 
+def _encode_to_jis_x_0201(value, errors='strict'):
+    """Convert a unicode string into JIX X 0201 byte string using shift_jis
+    encodings.
+    shift_jis is a superset of jis_x_0201. So we can regard the encoded value
+    as jis_x_0201 if it is single byte character.
+
+    Parameters
+    ----------
+    value : text type
+        The unicode string as presented to the user.
+    errors : str
+        The behavior of a character which could not be encoded. If 'strict' is
+        passed, raise an UnicodeEncodeError. If others are passed, replace
+        illegal character to '?'.
```

Comment:
In the case of passing 'replace' to encode, the illegal characters are replaced by the ASCII '?' (\x3f). Please see the [reference](https://docs.python.org/3.6/howto/unicode.html?highlight=unicode%20howto#converting-to-bytes)

So I use the following sentence.

"If any other value is passed, illegal characters are replaced by the ASCII '?'."

What do you think about it ?

Reply:
Ah, you are right - the other one is used for `decode()`, not for `encode()` - I have been confused, my bad! Maybe you can add a small test where this is shown?

Reply:
I got it. I'll add a small test for this feature.

Reply:
I found that the fallback didn't work as I expected by adding a small test. So I modified to replace all non ISO IR 14 characters into ASCII '?'. So even if any values whose charactercode is ISO IR 13 are given, we replace them. Because have to encode them without escape sequence. I think that it is not a critical issue.  Because this case occurs as a fallback.

Improve docstring:
https://github.com/pydicom/pydicom/blob/cedde3962e03aa174ea1d2d5428c1eebd7063544/pydicom/charset.py#L109

Improve fallback process:
https://github.com/pydicom/pydicom/blob/cedde3962e03aa174ea1d2d5428c1eebd7063544/pydicom/charset.py#L131

Add a test:
https://github.com/pydicom/pydicom/blob/cedde3962e03aa174ea1d2d5428c1eebd7063544/pydicom/tests/test_charset.py#L432